src/conditionals/programs_with_decision_points/medium/q6/Gamer.java

```diff
@@ -0,0 +1,34 @@
+package conditionals.programs_with_decision_points.medium.q6;
+
+/*
+Write a program that takes a directional input from the user, either 'w', 'a', 's', or 'd' and
```

Comment:
The question should also specify that the letters are lowercase just to be clear.

Reply:
Understandable